mobius-core/src/main/java/com/spotify/mobius/SafeConnectable.java

```diff
@@ -44,14 +44,16 @@
   @Nonnull
   @Override
   public Connection<F> connect(Consumer<E> output) {
-    final SafeConsumer<E> safeEventConsumer = new SafeConsumer<>(checkNotNull(output));
-    final Connection<F> effectConsumer =
-        new SafeEffectConsumer<>(checkNotNull(actual.connect(safeEventConsumer)));
-    final Disposable disposable = CompositeDisposable.from(safeEventConsumer, effectConsumer);
+    final SafeConsumer<E> safeConsumer = new SafeConsumer<>(checkNotNull(output));
+    final Connection<F> safeConnection =
```

Comment:
The new names are less clear, especially since both the `Safe` classes are nearly identical. It would be better to keep the `event` and `effect` parts, so `safeEventConsumer` and `safeEffectConnection`, I think.

Reply:
they aren't necessarily events or effects though, so i don't think this naming is a good idea to keep around.